block/src/Pos/DB/Block.hs

```diff
@@ -91,22 +93,31 @@ getTipBlock = getTipSomething "block" getBlock
 ----------------------------------------------------------------------------
 
 -- Get serialization of a block with given hash from Block DB.
-getSerializedBlock
-    :: forall ctx m. (HasConfiguration, MonadRealDB ctx m)
-    => HeaderHash -> m (Maybe ByteString)
-getSerializedBlock = blockDataPath >=> getRawData
+getSerializedBlock :: (MonadReader ctx m, HasLens NodeDBs ctx NodeDBs, MonadIO m, MonadCatch m) => FilePath -> HeaderHash -> m (Maybe ByteString)
+getSerializedBlock bdd hh = do
+  path <- blockDataPath hh
+  maybeContents <- getRawData path
+  pure maybeContents
 
 -- Get serialization of an undo data for block with given hash from Block DB.
 getSerializedUndo :: (HasConfiguration, MonadRealDB ctx m) => HeaderHash -> m (Maybe ByteString)
 getSerializedUndo = undoDataPath >=> getRawData
 
 -- For every blund, put given block, its metadata and Undo data into
 -- Block DB. This function uses 'MonadRealDB' constraint which is too
--- severe. Consider using 'dbPutBlund' instead.
+-- severe.
```

Comment:
Why the removal of the comment?

Reply:
the function it refers to was removed ages ago